Tests/GirdersSwiftTests/swift/http/TestRequest.swift

```diff
@@ -42,6 +42,26 @@ class TestRequest: XCTestCase {
         XCTAssertNotNil(request)
     }
     
+    func testRequestWithHeaderFields() {
```

Comment:
Write another test without additionalHeaderFields to test that nothing will be added.